src/Ninja/G1CP/Content/Misc/world.d

```diff
@@ -257,3 +257,19 @@ func int G1CP_GetWorldTime() {
 
     return hour * 100 + min;
 };
+
+/*
+ * Find VOB in the world by its exact position and move it to another position. Returns true on success.
+ *
+ * Simplifies fix template for "changeVobLocation"
+ */
+func int G1CP_ChangeVobLocation(var int originalPosPtr, var int correctedPosPtr, var int callback) {
```

Comment:
I'd like to change the name of this function to relate better to the existing find and move function and to stick to the suffix of "Pos", "PosPtr", and "PosF". Both will help in usage without having to look into the implementation - once familiar with the suffixes.

I suggest `G1CP_FindAndMoveVobToPosPtr(int posPtr, int movPtr, int callback)` and moving it just below the `G1CP_MoveVobToPosF` function to group the VOB position manipulation functions.

Reply:
The function is a bit atypical compared to the others in that file, because it has a return type (which **must not** be left unpopped). If we end up making more functions maybe we can consider grouping them in another file. For now this seems good.

Reply:
Following our fix templates, I would name this `G1CP_Tpl_ChangeVobLocation()` and move it to the templates file.

Reply:
That's a good suggestion. Let's go with that. When only used in the fixes anyway, the usage will be clear from code provided by the template.